sdk/core/core/test/curl_easy_perform_poc.c

```diff
@@ -47,12 +47,15 @@ int main() {
   }
 
   // Add auth
-  az_client_secret_credential credential = { 0 };
+  az_client_secret_credential credentials = { 0 };
+  uint8_t const * TENANT_ID = getenv("tenant_id");
+  uint8_t const * CLIENT_ID = getenv("client_id");
+  uint8_t const * CLIENT_SECRET = getenv("client_secret");
   az_result const creds_retcode = az_client_secret_credential_init(
-      &credential,
```

Comment:
Make sure to remove/rotate the secrets from Azure since these are now compromised